IFC4x3/Properties/t/Type_2yd82GST551f428rzb7p5q/DocProperty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Type_2yd82GST551f428rzb7p5q" Name="Type" UniqueId="bc9c8090-71d1-4506-9102-235f651f3174" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
```

Comment:
Propose renaming to `SparkGapType`.

Reply:
Agree. Commit : b5e658cad21e4944423ed95d945268c722299c68